lib/Completion/Bridge/TolerantParser/WorseReflection/WorseClassMemberCompletor.php

```diff
@@ -76,10 +76,10 @@ public function complete(Node $node, TextDocument $source, ByteOffset $offset):
         $reflectionOffset = $this->reflector->reflectOffset($source, $memberStartOffset);
 
         $symbolContext = $reflectionOffset->symbolContext();
-        $type = $symbolContext->type();
+        $symbolType = $symbolContext->type();
```

Comment:
would be idimatic to just call this `type` i think is it ambiguous?

Reply:
I changed this because $type is used in the loop `as`. It could be a bug if it at some point is being used after the foreach.